pkg/features/kube_features.go

```diff
@@ -884,8 +884,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	InTreePluginAzureFileUnregister:                {Default: false, PreRelease: featuregate.Alpha},
 	CSIMigrationvSphere:                            {Default: false, PreRelease: featuregate.Beta}, // Off by default (requires vSphere CSI driver)
 	InTreePluginvSphereUnregister:                  {Default: false, PreRelease: featuregate.Alpha},
-	CSIMigrationOpenStack:                          {Default: true, PreRelease: featuregate.Beta},
-	InTreePluginOpenStackUnregister:                {Default: false, PreRelease: featuregate.Alpha},
+	CSIMigrationOpenStack:                          {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.26
+	InTreePluginOpenStackUnregister:                {Default: true, PreRelease: featuregate.Beta},
```

Comment:
I think we still need the in-tree volume plugin to support kubelet 2 versions older than KCM. KCM must be able to use the in-tree volume plugin to attach/detach volumes to/from these old kubelets. I am afraid this can be on by default only when CSIMigrationOpenStack is GA + 2 releases.

Reply:
@jsafrane We still need line 887 to bring CSIMigrationOpenStack to GA in 1.24, right?
Your comment only refers to line 888 which should not be changed?

Reply:
thanks @xing-yang i reverted the change in line 888